schemas/qobj_schema.json

```diff
@@ -1134,6 +1134,24 @@
                                     }
                                 }
                             }
+                        },
+                        "qubit_lo_freq": {
```

Comment:
Same problem as above.

Reply:
Addressed in https://github.com/Qiskit/ibm-quantum-schemas/pull/9/commits/16396d9b899f20e78935a1904fc62314a6108514.